Targets/Mashow/Sources/DrinkSelection/View/DrinkTypeViewController.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  DrinkTypeViewController.swift
+//  Mashow
+//
+//  Created by ZENA on 8/5/24.
+//  Copyright © 2024 com.alcoholers. All rights reserved.
+//
+
+import UIKit
+import Combine
+import SnapKit
+
+final class DrinkTypeViewController: UIViewController {
+    
+    let viewModel: DrinkSelectionViewModel!
+    var drinkType: DrinkSelectionViewModel.DrinkType
+    var cancellables = Set<AnyCancellable>()
+    
+    init(viewModel: DrinkSelectionViewModel, drinkType: DrinkSelectionViewModel.DrinkType) {
+        self.viewModel = viewModel
+        self.drinkType = drinkType
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private lazy var typeTitleImageView: UIImageView = {
+        let image = UIImage(named: drinkType.rawValue)
+        let imageView = UIImageView(image: image)
+        return imageView
+    }()
+    
+    private lazy var addDrinkTypeButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 0, y: 0, width: 127, height: 40))
+        button.layer.cornerRadius = 20
+        button.layer.masksToBounds = true
+        
+        let view = UIStackView()
```

Comment:
```suggestion
        let stackView = UIStackView()
```
개 쪼잘한 건데 이렇게 써주시면 좀 보기 편할듯..ㅎ

Reply:
➕  이것도 조잘한 건데
<img width="764" alt="image" src="https://github.com/user-attachments/assets/3daaad97-1a8d-4ac3-8365-459fde0ec52f">

요런식으로 뷰-뷰 수정 pair로 코드를 묶어 써주시면 가독성이 좀 더 좋아집니다..ㅎ

Reply:
하핫 머쓱 고맙습니다
여기 잘 안돼서 이것저것해보다보니 ~~